polymorphic/query_translate.py

```diff
@@ -162,7 +162,7 @@ def translate_polymorphic_field_path(queryset_model, field_path):
         # so no tripple ClassName___field was intended.
         try:
             # rel = (field_object, model, direct, m2m)
-            field = queryset_model._meta.get_field_by_name(classname)[0]
+            field = queryset_model._meta.get_field(classname)
```

Comment:
No, at least tests don't reflect it. Doc doesn't say that this is new to 1.8 either.